dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/messages/messages.properties

```diff
@@ -147,8 +147,8 @@ UpdateCodewindDialogTitle=Codewind Update
 UpdateCodewindDialogMsg=This will remove the older version of Codewind and install, and start the newer version of Codewind. Do you want to continue?
 
 SelectProjectTypePageName=Select Project Type
-SelectProjectTypePageTitle=Project Type and Language Selection
-SelectProjectTypePageDescription=Select a project type. If the type supports multiple languages, optionally select a language.
+SelectProjectTypePageTitle=Project Type Selection
+SelectProjectTypePageDescription=Select a project type
```

Comment:
Please add a period at the end of this sentence. :)

Reply:
I left it out, because I checked in most other wizards the description does not have ending period

Reply:
OK, if other Eclipse wizards don't have the period, we can keep it consistent. 👍